localtypings/pxtpackage.d.ts

```diff
@@ -73,6 +73,7 @@ declare namespace pxt {
         weight?: number;
         gistId?: string;
         extension?: PackageExtension; // describe the associated extension if any
+        isExtension?: boolean; // is this package an extension
```

Comment:
who is supposed to set this flag? is there a UI for it?
what about the existing extension, do they have to be updated?

Reply:
This flag gets set automatically in the toolbox code!